README.md

```diff
@@ -340,7 +338,7 @@ log-derived map (VLDM), as described in the
 [Verifiable Data Structures](docs/VerifiableDataStructures.pdf) white paper
 (which uses the term 'log-backed map').  To do this, a VLDM personality would
 monitor the additions of entries to a Log, potentially external, and would write
-some kind of corresponding key:value data to a Trillian Map.
+some kind of the corresponding key:value data to a Trillian Map.
```

Comment:
this sentence doesn't make sense now, I think it was fine the way it was previously.